Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php

```diff
@@ -77,4 +55,52 @@ public function getDescription()
     {
         return 'Binary operators should be surrounded by at least one space.';
     }
+
+    private function fixWhiteSpaceAroundOperator(Tokens $tokens, $index)
+    {
+        // do not change the alignment of `=>` or `=`, see `(un)align_double_arrow`, `(un)align_equals`
```

Comment:
I pref. this way because it would restores behavior to the previous way the fixer worked. 
Adding configuration would make the fixer more complex without much benefits. This is because on 1.x you can create the same result using the alignment fixers; the configuration will not be portable 1-0-1 for 2.x line so people will need to update their configuration anyway.
My suggestion is to merge this PR to 1.x and than I make a master version which will use configuration options.